testsuite/replay_verify.py

```diff
@@ -52,7 +52,8 @@ def replay_verify_partition(
     # run and print output
     process = subprocess.Popen(
         [
-            "target/release/replay-verify",
+            "target/release/aptos-db-tool",
```

Comment:
while you're at it, can you change this line from `testsuite/replay-verify.py` to `testsuite/replay_verify.py` (- to _)? Replay verify is supposed to run on PRs that touch this file, but there's a typo that's preventing it from being triggered on PR:

https://github.com/aptos-labs/aptos-core/blob/main/.github/workflows/replay-verify.yaml#L24

Reply:
updated the file name

Reply:
Looks like that caught a bug: https://github.com/aptos-labs/aptos-core/actions/runs/4239339946/jobs/7367299441

Reply:
add this new option that wasn't included in the db-tool previously

Reply:
> testsuite/replay-verify.py to testsuite/replay_verify.py

my bad and thank you